src/main/rx/rx.c

```diff
@@ -204,6 +203,13 @@ static uint8_t nullFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
     return RX_FRAME_PENDING;
 }
 
+static bool nullProcessFrame(rxRuntimeConfig_t *rxRuntimeConfig)
+{
+    UNUSED(rxRuntimeConfig);
+
+    return true;
+}
+
```

Comment:
```
static void nullProcessFrame(rxRuntimeConfig_t *rxRuntimeConfig)
 {
     rxRuntimeConfig-> processingRequired = false;
 }
```

Reply:
Don't know about that one. To me it is 'cleaner' if the handler functions do not modify the global state directly, but instead of that communicate through return values.

If we do this change, should we also change `rcFrameStatusFn` to directly modify the state instead of using return values, for consistency's sake?

Reply:
On second thoughts, I agree with you. It is cleaner to use return values.

In that case function signature should be `static void nullProcessFrame(const rxRuntimeConfig_t *rxRuntimeConfig)`

(And we should probably do a new PR to make `rxRuntimeConfig` const for the `rcFrameStatusFnPtr` functions as well.

Reply:
Wouldn't that make it `static bool nullProcessFrame(const rxRuntimeConfig_t *rxRuntimeConfig)`?

Reply:
Yes, sorry. `static bool...`